microsoft-365/commerce/subscriptions/allowselfservicepurchase-powershell.md

```diff
@@ -37,10 +37,10 @@ You can use the **MSCommerce** PowerShell module to:
 
 To use the **MSCommerce** PowerShell module, you need:
 
-- A Windows 10 device
+- A Windows 10 or later operating system.
 - PowerShell 5 or below. Currently, PowerShell 6.x/7.x isn't supported with this module.
-- Administrator permission for the device
-- Global or Billing Admin role for your tenant
+- Global or Billing admin role for your tenant to change the MSCommerce Product Policies.
```

Comment:
Change to:
The Global or Billing admin role for your tenant to change the **MSCommerce** product policies.